### DISCUSSION TITLE
Can this tool run as a service?

### DISCUSSION THREAD

**Discussion by ananan** [projectdiscovery/nuclei#960] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
nuclei is a great tool for security research, but for now it can only run in command line, for some case we need this tool run as a daemon service so that we can push targets to scan continually.  

**Describe the solution you'd like**
nuclei can run as a daemon service, thus we can push targets to scan and get the results instantly, and easily intergrated with other component.

**Comment by ehsandeep** [comment, marked as answer]
Hi @ananan, 

Yes, that's planned for Nuclei v3, you will be able to run nuclei as a self-hosted service and initiate REST API-based scans.

**Reply by ananan** [reply]
Thanks, may I ask when?

**Reply by Ovi3** [reply]
Hi @ehsandeep ,
I've been having similar thoughts about this matter and was wondering if there have been any progress.
